Gems/MotionMatching/Code/Source/MotionMatchingData.h

```diff
@@ -41,6 +41,12 @@ namespace EMotionFX::MotionMatching
         MotionMatchingData(const FeatureSchema& featureSchema);
         virtual ~MotionMatchingData();
 
+        enum EMFX_API FeatureScalerType
```

Comment:
question: is `EMFX_API` needed here for the enum?

Reply:
Removed